src/styling/navbar.css

```diff
@@ -1,4 +1,89 @@
+:root {
+  --black-2: #121212;
+  --grey-2: hsl(0, 0%, 91%);
+  --white: #fff;
+  --sb: space-between;
+}
+
 .H-con {
```

Comment:
- [ ] As it is a requirement for this project and a best practice for a front-end dev, to stick to the design, please make sure you `navbar` aligns with the page. Checkout the images below and fix the margins. 

***The design***
![hamidD](https://user-images.githubusercontent.com/65852621/145552135-451e0822-4cad-4670-a3e8-3433c18b51e2.png)

***Your implementation***
![hamidI](https://user-images.githubusercontent.com/65852621/145552204-15f744ab-e509-4d75-a834-9a6021a8a796.png)